apps/web/src/components/Filters/RollupFilter.tsx

```diff
@@ -0,0 +1,32 @@
+import type { FC } from "react";
+
+import { Rollup as Rollups } from "@blobscan/api/enums";
+
+import { Dropdown } from "~/components/Dropdown";
+import type { DropdownProps } from "~/components/Dropdown";
+import { RollupIcon } from "~/components/RollupIcon";
+import type { Rollup } from "~/types";
+import { capitalize } from "~/utils";
+
+type RollupFilterProps = Pick<DropdownProps, "onChange" | "selected">;
+
+export const RollupFilter: FC<RollupFilterProps> = function ({
+  onChange,
+  selected,
+}) {
+  return (
+    <>
+      <Dropdown
+        selected={selected}
+        options={Object.values(Rollups).map((rollup) => ({
+          value: rollup.toLowerCase(),
+          label: capitalize(rollup),
+          prefix: <RollupIcon rollup={rollup.toLowerCase() as Rollup} />,
+        }))}
```

Comment:
I forgot to mention this in the Dropdown refactor PR: I believe the dropdown’s `prefix` prop is unnecessary now that we can pass a React node as the label. We can simply include the prefix directly in the label component.

Could we instead have a label that includes both the `RollupIcon` component and the capitalized rollup name? This way, the selected rollup would be rendered with the icon, as opposed to the current implementation where only the text appears.

![image](https://github.com/user-attachments/assets/f069ee26-ee69-438a-b4f0-50eb13c3feb1)

Reply:
I'd use drop the `prefix` prop as well from the dropdown component